package.json

```diff
@@ -29,7 +29,7 @@
     "release": "release-it"
   },
   "devDependencies": {
-    "@appuniversum/ember-appuniversum": "~2.15.0",
+    "@appuniversum/ember-appuniversum": "~3.4.1",
```

Comment:
Personally I think we can go for `^3.4.1` as AU versions follow semver pretty well.